src/transport/raw/MessageHeader.cpp

```diff
@@ -158,11 +160,22 @@ CHIP_ERROR PacketHeader::Decode(const uint8_t * const data, uint16_t size, uint1
     err = reader.Read32(&mMessageId).StatusCode();
     SuccessOrExit(err);
 
+#ifdef DEBUG
+    ChipLogDetail(Inet, "PacketHeader::Decode");
+    ChipLogDetail(Inet, "  header=%u", (int) header);
+    ChipLogDetail(Inet, "  mEncryptionType=%d", (int) mEncryptionType);
+    ChipLogDetail(Inet, "  mFlags=%d", mFlags.Raw());
+    ChipLogDetail(Inet, "  mMessageId=%d", mMessageId);
+#endif
```

Comment:
Why ifdef DEBUG?  If we're doing detail logging anyway, fine.  Let's just do it.

Reply:
addressed in latest